applications/nrf5340_audio/src/bluetooth/ble_audio_services.c

```diff
@@ -187,14 +191,19 @@ static void mcc_discover_mcs_cb(struct bt_conn *conn, int err)
 
 	LOG_DBG("mcc_discover_mcs_cb");
 
+	if (!mcs_discover_called) {
+		return;
+	}
```

Comment:
Consider calling `ble_mcs_discover_called_set(false)` in this callback as well

Reply:
Done